src/components/search/search.stories.mdx

```diff
@@ -177,6 +177,101 @@ In this example the `variant` prop is "dark" and no search button is present.
   </Story>
 </Canvas>
 
+### Validations
+
+Validation status can be set by passing `error`, `warning` or `info` prop to the component
+
+Passing a string to these props will display a properly colored border along with a validation icon and tooltip - string value will be displayed as the tooltip message.
+
+Passing a boolean to these props will display only a properly colored border.
+
+For more information check our [Validations](?path=/docs/documentation-validations--page "Validations") documentation page
+
+#### As a string
+
+<Canvas>
+  <Story name="validations - string - component">
+    {() => {
+      const [state, setState] = useState({
+        error: "<>/",
+        warning: "<>/",
+        info: "<>/",
+      });
+      const handleChange = (validation) => (e) => {
+        setState({ ...state, [validation]: e.target.value.rawValue });
```

Comment:
```suggestion
        setState({ ...state, [validation]: e.target.value });
```

Not sure what the `rawValue` is doing here - the component is crashing for me with this in, and `e.target.value` is just the string value. (And this matches the "controlled" story above.)

Reply:
my mistake, I'll remove that